src/test/java/lotto/domain/LottoCashierTest.java

```diff
@@ -0,0 +1,32 @@
+package lotto.domain;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class LottoCashierTest {
+
+    private final LottoCashier lottoCashier = new LottoCashier();
+
+    @ParameterizedTest
+    @CsvSource(value = {"1000,1", "1500,1"})
+    void pay(int money, int lottoCount) {
+        List<Lotto> lottos = lottoCashier.pay(money);
+        assertThat(lottos.size()).isEqualTo(lottoCount);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {0, 500})
+    void payFail(int money) {
+        assertThatThrownBy(() -> {
+            lottoCashier.pay(money);
+        }).isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("해당 금액으로는 로또를 사실 수 없습니다.");
+    }
```

Comment:
람다 표현식을 통해 조금 더 가독성이 좋아질 수 있을 것 같아요 😄 
```suggestion
    @ParameterizedTest
    @ValueSource(ints = {0, 500})
    void payFail(int money) {
        assertThatThrownBy(() -> lottoCashier.pay(money))
                .isInstanceOf(IllegalArgumentException.class)
                .hasMessage("해당 금액으로는 로또를 사실 수 없습니다.");
    }
```

Reply:
줄 정렬이 어색했었는데 보기 훨씬 깔끔해졌네요!